breacharbiter.go

```diff
@@ -372,7 +371,8 @@ func (b *breachArbiter) breachObserver(contract *lnwallet.LightningChannel,
 		// breached in order to ensure any incoming or outgoing
 		// multi-hop HTLCs aren't sent over this link, nor any other
 		// links associated with this peer.
-		b.htlcSwitch.CloseLink(chanPoint, CloseBreach)
+		b.server.CloseChannel(chanPoint, closeBreach)
```

Comment:
Yep, you are right, after additional review I understood that now there is no need in such movement, before I thought that without it decoupling of `htlcsSwitch` would be impossible. Returned logic back.